dashboard.el

```diff
@@ -180,6 +163,78 @@ If MESSAGEBUF is not nil then MSG is also written in message buffer."
     (beginning-of-line)
     (widget-forward 1)))
 
+(defun dashboard-generate-banner ()
+  (when (goto-char (point-max))
+    (dashboard-insert-banner)
+    (dashboard-insert-page-break)))
+(defun dashboard-generate-recents ()
+  (recentf-mode)
+  (when (dashboard-insert-file-list
+	 "Recent Files:"
+	 (dashboard-subseq recentf-list 0 list-size))
+    (dashboard-insert--shortcut "r" "Recent Files:")
+    (dashboard-insert-page-break)))
+(defun dashboard-generate-bookmarks ()
+  (require 'bookmark)
+  (when (dashboard-insert-bookmark-list
+	 "Bookmarks:"
+	 (dashboard-subseq (bookmark-all-names)
+			   0 list-size))
+    (dashboard-insert--shortcut "m" "Bookmarks:")
+    (dashboard-insert-page-break)))
+(defun dashboard-generate-projects ()
```

Comment:
To be honest, i don't like the `generate` name. Since the call is `dashboard-insert-startupify-list` i would go for `insert`